sdk/core/Azure.Core/perf/DownloadTest.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core.Pipeline;
+using Azure.Identity;
+using Azure.Test.Perf;
+
+namespace Azure.Template.Perf
+{
+    public class DownloadTest : PerfTest<DownloadTestOptions>
+    {
+        // Stream.CopyToAsync default buffer
+        private const int BufferSize = 81920;
```

Comment:
Why does `DownloadTest` need a `BufferSize`, but `DownloadHttpClientTest` does not?

Reply:
HttpClient doesn't expose the response stream directly. Instead it has a CopyTo method that doesn't expose a way to set the buffer.